staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go

```diff
@@ -84,7 +84,15 @@ func (c *azureTokenCache) setToken(tokenKey string, token *azureToken) {
 	c.cache[tokenKey] = token
 }
 
+var warnOnce sync.Once
+
 func newAzureAuthProvider(_ string, cfg map[string]string, persister restclient.AuthProviderConfigPersister) (restclient.AuthProvider, error) {
+	// deprecated in v1.22, remove in v1.25
+	warnOnce.Do(func() {
+		klog.Warningf(`WARNING: in-tree azure auth plugin is now deprecated.
+Please use the https://github.com/Azure/kubelogin kubectl/client-go credential plugin instead.`)
```

Comment:
As a casual/novice Kubectl user `kubectl/client-go` doesn't seem very actionable to me (as opposed to Azure's login thing getting an entire URL). Is it a directory? Or package? Or Repo? I would prefer a more actionable link where I can learn how to use `kubectl/client-go`.

Reply:
suggestions for alternative links are welcome

Reply:
Added a link to the credential plugin docs.